docs/_documentations/remotedeploy-eclipse.md

```diff
@@ -12,31 +12,33 @@ order: 2
 
 ## Connecting Eclipse to Remote Codewind
 
-Locate the Codewind view in Eclipse and click the cloud icon to launch the new connection wizard:
+1\. Locate the Codewind view in Eclipse and click the cloud icon to launch the new connection wizard:
 
 ![New Connection](./images/remoteeclipse/newConnection.png)
 
-Add a connection name, connection URL of your deployment, username and password
+2\. Add a connection name, the connection URL of your deployment, username, and password:
 
 ![Required Fields](./images/remoteeclipse/connectionCreds.png)
 
-The IDE will then validate the connection and add it to the Codewind panel:
+The IDE validates the connection and adds it to the Codewind panel:
 
 ![Validate settings](./images/remoteeclipse/connectionAdded.png)
 
-At this the IDE and Codewind are connected
+The IDE and Codewind are connected.
 
 ## Adding a deployment registry
 
-Before projects can be deployed on Kubernetes you need to specify a docker registry. In this example we will use DockerHub.  If its not already open navigate to the connection details page by right clicking the Deployment and selecting Manage Deployment Registries:
+Before projects can be deployed on Kubernetes, you must specify a docker registry. This example uses DockerHub. 
+
+1\. If it's not already open, navigate to the connection details page; right-click the deployment and select `Manage Deployment Registries`:
```

Comment:
I think the instructions would be a little clearer if the sentence became two or if it were a list: "If it's not already open, navigate to the connection details page, right-click the deployment, and select..."